internal/service/credentials/resource_credential_type.go

```diff
@@ -161,18 +173,40 @@ func (r *CredentialTypeResource) Schema(ctx context.Context, req resource.Schema
 			),
 
 			"title": schema.StringAttribute{
-				Description: "Title of the credential. Verification sites are expected to be able to request the issued credential from the compatible wallet app using the title.",
-				Required:    true,
+				Description:         titleDescription.Description,
+				MarkdownDescription: titleDescription.MarkdownDescription,
+				Required:            true,
 				Validators: []validator.String{
 					stringvalidator.LengthAtLeast(attrMinLength),
+					customstringvalidator.IsRequiredIfRegexMatchesPathValue(
+						regexp.MustCompile(`\${cardTitle}`),
+						"The title argument is required because the ${$cardTitle} element is defined in the card_design_template.",
```

Comment:
`${$cardTitle}` should probably be `${cardTitle}`

Reply:
`card_design_template` should probably be in MD quotes

Reply:
fixed